backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+    finally:
+        db.close()
+
+UPLOAD_DIR = "uploads"
+os.makedirs(UPLOAD_DIR, exist_ok=True)  # Ensure upload directory exists
+
+
+@router.post("/step1", response_model=UserResponse)
+def complete_profile_step1(
+    university_name: str = Form(...),
+    department: str = Form(...),
+    fields_of_interest: List[str] = Form(...),  # ✅ Corrected list input
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)